web/src/compose_recipient.ts

```diff
@@ -247,6 +247,7 @@ function item_click_callback(event: JQuery.ClickEvent, dropdown: Instance): void
     }
     compose_state.set_selected_recipient_id(recipient_id);
     on_compose_select_recipient_update();
+    compose_state.set_recipient_edited_manually(true);
```

Comment:
Should this be before `on_compose_select_recipient_update`? It seems logically like this sort of edit-state-values should happen before hooks.

Reply:
I haven't seen bugs from it being in this order, but swapping the order to what you described makes sense to me so I'll do that.